Formula/node.rb

```diff
@@ -56,30 +56,11 @@ def install
 
     if build.with? "npm"
       resource("npm").stage buildpath/"npm_install"
-
-      # make sure npm can find node
-      ENV.prepend_path "PATH", bin
-      # set log level temporarily for npm's `make install`
-      ENV["NPM_CONFIG_LOGLEVEL"] = "verbose"
-      # unset prefix temporarily for npm's `make install`
-      ENV.delete "NPM_CONFIG_PREFIX"
-
-      cd buildpath/"npm_install" do
-        system "./configure", "--prefix=#{libexec}/npm"
```

Comment:
I'm not sure how much more @chrmoritz and I can get into this really 😕. At some point I think I'm kind of asking folks that as the two people primarily responsible for keeping Node as stable as it has been for the last 2+ years you trust that we know what's going on, why & why it isn't unreasonable here.

I'm kind of at a point where I'm going look this works, it's behaving as upstream expected, Homebrew has always had to hack around `npm` to play nice with Homebrew, this is the latest version of that. You can choose not to take my word on this but at some point I'm going to turn around and say I've done all I can reasonably do here & pass it off onto one of you two to resolve.

I'm doing my best here but we're reaching the point where my/Christian's understanding of `npm` is getting rolled under a bus & we're looping round to the same arguments, which increasingly are taking the tone that upstream don't intend or don't understand the way their own tool should be behaving.

I'm trying my best folks, but I'm running out of ways to make y'all happy.